packages/workbox-build/test/node/get-file-manifest-entries.js

```diff
@@ -263,38 +263,38 @@ describe('Test getFileManifestEntries', function() {
       globDirectory: path.join(__dirname, '..', '..', '..',
         'workbox-cli', 'test', 'static', 'example-project-1'),
       dynamicUrlToDependencies: {
-        '/template/url1': ['page-1.html', 'index.html'],
```

Comment:
With the fix where all leading "/" are removed we need to ensure all the tests also do not point to root "/'. If the above tests are not updated then it errors out during the run.... 

for e.g. without the change to dynamicUrlToDependencies we get the following

```
  7) Test Runner Environment  Node Tests for workbox-build Test getFileManifestEntries should return file entries from example project with dynamicUrlToDependencies:

      AssertionError: expected [ Array(8) ] to deeply equal [ Array(8) ]
      + expected - actual

       [
         {
      -    "revision": "66f6a91fb924cf7e0b59d496a229c08a"
      +    "revision": "24abd5daf6d87c25f40c2b74ee3fbe93"
           "url": "index.html"
         }
         { /* Deleted some stuff here to keep sample cleaner */    }
         {
      -    "revision": "b7ccb766d7273d9c0095706bbce65a6e"
      -    "url": "/template/url1"
      +    "revision": "a505dfb0ac2cad8933ec437dd97ccc66"
      +    "url": "template/url1"
         }
         {
      -    "revision": "76d943bf4ee4fb547a7dd541464618ff"
      -    "url": "/template/url2"
      +    "revision": "bd9ef0ab8b57d5d716e6916610d34936"
      +    "url": "template/url2"
         }
         {
           "revision": "538954a0f0fca1d067ff03dca8dce79e"
      -    "url": "/template/url3"
      +    "url": "template/url3"
         }
       ]

```